dev-docs/bidders/oneplanetonly.md

```diff
@@ -13,11 +13,10 @@ prebid_1_0_supported : true
 
 ### bid params
 
-| Name   | Scope    | Description | Example         |
-| :---   | :----    | :---------- | :------         |
-| siteId   | required | The site id     | '5'     |
-| adUnitId | required | The ad unit id  | '5-4587544'     |
-
+| Name     | Scope    | Description    | Example       | Type     |
```

Comment:
Another one that's not displaying correctly. Looks like this one needs `{: .table .table-bordered .table-striped }` to format the table. (Not your mistake, looks like it was never there.)